### DISCUSSION TITLE
register volt package component

### DISCUSSION THREAD

**Discussion by jdion84** [livewire/livewire#7701] [1. Help]
how would i register a volt component in a package service provider?

for example, with traditional class-based components i can do:

```php
Livewire::component('some-component', SomeComponent::class);
```

**Comment by tomshaw** [comment]
Running into same issue. In my package service provider I've registered where to load the views.

```php
$this->loadViewsFrom(__DIR__ . '/../../resources/views', 'mypackage');
```

Yet unable to load:

```html
<livewire:mypackage::components.header :$data />
```
